lightning/src/util/message_signing.rs

```diff
@@ -0,0 +1,90 @@
+use crate::util::zbase32;
```

Comment:
All public methods and modules need documentation - the `pub fn`s can probably get something simple, but the module here should describe the signing here, note that its a special extension of c-lightning and lnd and not a part of the spec, and maybe link to some documentation on it at c-lightning and/or lnd.

Reply:
Added docs at 3b52b18